includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -77,6 +80,90 @@ public function unregister_embed() {
 		remove_filter( 'embed_oembed_html', [ $this, 'filter_embed_oembed_html' ], 10 );
 	}
 
+	/**
+	 * Sanitize HTML that are not added via Gutenberg.
```

Comment:
This is somewhat incorrect as HTML produced by Gutenberg could also be handled by this method if WordPress didn't convert the URL into an embed, for example.

```suggestion
	 * Sanitize YouTube raw embeds.
```